semcore/input/src/Input.jsx

```diff
@@ -120,15 +120,21 @@ class Input extends Component {
 }
 
 class Value extends Component {
-  static defaultProps = {
-    defaultValue: '',
+  static defaultProps = (props) => {
+    return props.disableUncontrolledValueChange
```

Comment:
You don't need these changes in this file, isn't it?

Reply:
yes, removed